src/runtime/config/configuration-qemu-snp.toml.in

```diff
@@ -44,6 +44,9 @@ confidential_guest = true
 # enable SEV SNP VMs
 sev_snp_guest = true
 
+# Path of SNP certificates blob
+snp_certs_path = @SNPCERTSPATH@
```

Comment:
You might want to expand this comment to say what the default path will be. You could also mention that this is for the extended report and the VCEK/VLEK specifically.

Reply:
Sure, i will do that.